Vic2ToHoI4/Data_Files/blankMod/output/common/on_actions/05_lar_on_actions.txt

```diff
@@ -214,7 +214,6 @@ on_actions = {
 				if = {
 					limit = {
 						has_trait = operative_linguist
-						NOT = { operative_leader_mission = no_mission }
```

Comment:
Why was this removed? It's in the base game.

Reply:
This line cause a bug making the linguist operative trait do nothing at all.
I wanted to report that myself but then I found that somebody else made a forum post 4 months before indicating how to fix it in minute details (aka remove that line).